### PR TITLE
Adding APi V2 Support for Debezium

### PR DESCRIPTION
**Pull request by RussDavidson-InstaClustr**

### CODE REVIEW

**Review by instaclustr-bing — APPROVED** [review]

**Review by iclonghoang — APPROVED** [review]

**Review by iclonghoang — CHANGES_REQUESTED** [review]
can you re-gen docs after api specs update?

File: docs/data-sources/cassandra_cluster_v2_instance.md [hunk 1]
```diff
@@ -237,6 +240,22 @@ Settings to determine how resize requests will be performed for the cluster.<br>
...
+### Read-only attributes
+*___kafka_topic_prefix___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>Kafka Topic Prefix<br><br>
+*___kafka_cdc_id___*<br>
+<ins>Type</ins>: string (uuid), read-only<br>
+<br>Kafka Cluster Data Centre Id<br><br>
+*___version___*<br>
+<ins>Type</ins>: string, read-only<br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Debezium Connector Cassandra to the Cassandra cluster. --AVAILABLE_BUNDLE_VERSIONS_MARKER_V2_DEBEZIUM--<br><br>
```
Comment (iclonghoang) [inline docs/data-sources/cassandra_cluster_v2_instance.md:255]: `--AVAILABLE_BUNDLE_VERSIONS_MARKER_V2_DEBEZIUM--`

this is unexpected, can you test local API doc to see if versions are listed properly?
https://instaclustr.atlassian.net/wiki/spaces/IDS/pages/1074036743/Local+and+Preproduction+Testing+API+Documentation

Comment (RussDavidson-InstaClustr) [reply docs/data-sources/cassandra_cluster_v2_instance.md]: Hi Long,

Thanks for this, it should be fixed now.

Comment (iclonghoang) [reply docs/data-sources/cassandra_cluster_v2_instance.md]: nice,
just noticed a conflict in Makefile 
except that, everything is good

**Review by iclonghoang — APPROVED** [review]
pseudo approve to remove 'request for changes'

**Review by iclonghoang — APPROVED** [review]

**Review by instaclustr-bing — APPROVED** [review]

**Pull request merged**